intranet/templates/events/event.html

```diff
@@ -123,11 +123,6 @@ <h3>
         <div class="bottom-row show-attend">
             <span class="signup-status">
                 People attending: {{ event.attending.count }}
-
-                <a href="{% url 'event_roster' event.id %}" class="button small-button view-roster-button">
-                    View Roster
-                </a>
-
```

Comment:
I'm not understanding your logic here. You still want people to be able to see who is coming.

Reply:
Lines 15-17 already include a link to the roster:
` <a href="{% url 'event' event.id %}" class="event-link">{{ event.title }}</a>`
The /events/event/{id} URL path also renders the roster view (in addition to /events/roster), but I can add back the button if you would prefer that.